### PR TITLE
Features

### PR DESCRIPTION
**Pull request by Abel-desu**
Portfolio setup and mobile version skeleton

I added some features on it
-giving font-size
-updating css part

### CODE REVIEW

**Comment by BenMukebo** [comment]
@hey!!

**Review by BenMukebo — COMMENTED** [review]
## STATUS: INVALID ❌ 

Hi  @Abel-desu,

I can't miss felicitating your efforts and energy, you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request 

- [ ] You have submitted a project, that doesn't have a Pull Request.


### To improve ⏫ 

- [ ] Please, Write a meaningful title for your pull request and always add a short summary of changes introduced in the pull request. you can check the  [Professional repo - basic rules](https://github.com/microverseinc/curriculum-transversal-skills/blob/main/documentation/articles/professional_repo_rules.md)


Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me (@BenMukebo) in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **Does not count** into the code reviews limit.

**Comment by Abel-desu** [comment]
This is Portfolionsetup and mobile version skeleton project

**Comment by Abel-desu** [comment]
@BenMukebo check it now

File: index.html [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+      </div>
+        </div>
+        <div class=" social_links">
+          <div class="twit-icon"><img src="images/twitter.png" alt="twitter"></div>
+          <div class="git-icon"><img src="images/github.png" alt="github"></div>
+          <div class="ln-icon"><img src="images/Linkedin-icon.png" alt="Linkedin-icon"></div>
+          <div class="m-icon"><img src="images/medium.png" alt="medium"></div>
+          <div class="a-icon"><img src="images/angellist.png" alt="angellist"></div>
+
+        </div>
```
Comment (aimemalaika) [inline index.html]: - [ ] The list of icons uses a ul tag to group them, and li per icon; each icon has a link (a tag). Kindly work on that
- [ ] Kindly use flex to display the icons list

File: index.html [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+      
+        <div class="container">
+        <div class="section">
+        
+      <div class="mainsection">
+        <div class="maincontainer">
+      <div class="section1">
+        <h2 class="text1">
+          i'm Abel<br />
+          Glad to see you!</h2><br/>
```
Comment (aimemalaika) [inline index.html]: - [ ] kindly make sure that the main title (e.g., "I'm Marianne ...") is a `h1` tag

File: index.html [hunk 3]
```diff
@@ -0,0 +1,54 @@
...
+        
+        <h2 class="text2">
+          I'm a software developer! i can help you build a product, feature or website Look through someof my work and experience! if you like what you see and have you need coded, don't hesitate to contact me.
+        </h2>
```
Comment (aimemalaika) [inline index.html]: - [ ] Kindly make sure that  this paragraph (e.g., "I'm a software ...") is a `p` tag

File: index.html [hunk 4]
```diff
@@ -0,0 +1,54 @@
...
+        <div class="let">
+        <p class="text3">LET'S CONNECT</p>
```
Comment (aimemalaika) [inline index.html]: - [ ] Kindly make sure this is a button or a link.

File: index.html [hunk 5]
```diff
@@ -0,0 +1,54 @@
...
+  </head>
+  <body>
+      
+        <div class="container">
+        <div class="section">
+        
+      <div class="mainsection">
+        <div class="maincontainer">
+      <div class="section1">
+        <h2 class="text1">
```
Comment (aimemalaika) [inline index.html]: - [ ] Kindly make sure the header part of your portfolio should be in a `header` tag
- [ ] Kindly put the logo (if the design has one) is on the left and have a link (`a` tag)
- [ ] Kindly make sure the hamburger button is on the right and has a `nav` tag

**Review by aimemalaika — CHANGES_REQUESTED** [review]
Hi @Abel-desu, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

the-simpsons-homer-simpson

 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by OLIPLICHE — APPROVED** [review]
Hi @Abel-desu, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**